chain/types/src/lib.rs

```diff
@@ -229,24 +229,12 @@ impl<M: Middleware> From<&ContractInstances<M>> for ContractAddresses {
     }
 }
 
-fn workspace_dir() -> std::path::PathBuf {
-    let output = std::process::Command::new(env!("CARGO"))
-        .arg("locate-project")
-        .arg("--workspace")
-        .arg("--message-format=plain")
-        .output()
-        .unwrap()
-        .stdout;
-    let cargo_path = std::path::Path::new(std::str::from_utf8(&output).unwrap().trim());
-    cargo_path.parent().unwrap().to_path_buf()
-}
-
 /// Creates local Anvil instance.
 ///
 /// Used for testing. When block time is given, new blocks are mined periodically.
 /// Otherwise, a new block is mined per transaction.
 pub fn create_anvil(block_time: Option<std::time::Duration>) -> ethers::utils::AnvilInstance {
-    let mut anvil = ethers::utils::Anvil::new().path(workspace_dir().join(".foundry/bin/anvil"));
+    let mut anvil = ethers::utils::Anvil::new();
```

Comment:
Why?

Reply:
Moving PATH management outside of such source files makes these more robust.